suricata/update/configs/modify.conf

```diff
@@ -18,3 +18,7 @@
 # For compatibility, most Oinkmaster modifysid lines should work as
 # well.
 # modifysid * "^drop(.*)noalert(.*)" | "alert${1}noalert${2}"
+
+# Add metadata.
+#metadata-add re:"SURICATA STREAM" "evebox-action" "archive"
+#metadata-add 2010646 "evebox-action" "archive"
```

Comment:
The intended behavior is most recent metadata be added only and not a list of metadata, is that correct?
e.g.
```
metadata-add re:"SURICATA STREAM" "evebox.action" "archive"
metadata-add re:"SURICATA STREAM" "evebox.action" "delete"
```
there should only be `metadata: [evebox-action delete]` in the matching rules?

Asking because it showed up as a list in the rule after `suricata.update.rule.parse`..